tensorflow/lite/delegates/gpu/delegate.cc

```diff
@@ -334,8 +332,10 @@ class DelegateKernel {
     enforce_same_thread_ = true;
     TFLITE_LOG_PROD_ONCE(tflite::TFLITE_LOG_INFO,
                          "Initialized OpenGL-based API.");
-#endif
     return absl::OkStatus();
+#else
+    return absl::UnavailableError("OpenGL-based API disabled");
```

Comment:
Is this needed since we only support OpenCL?

Reply:
Yes. Without this fix default behaviour is crash in case opencl failed to initialize.